test/test_helper.rb

```diff
@@ -70,8 +70,8 @@ def login_platform_admin
 
   def create_store_admin
     store = Store.create(name: "Dave's Cars")
-    admin = User.create(username: 'storeadmin',
-                        password: 'password')
+    admin = User.create(username: 'Dave',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.